### ISSUE TITLE
Ausblenden von terminen in der vergangenheit

### ISSUE THREAD

**Issue by Roland-Augsburg** [wzolleis/djk-kicker#22]

**Comment by wzolleis** [comment]
das sollte mit den zwei Buttons "alle Spiele" bzw. "aktuelles Spiel" möglich sein.
Wir haben auch schon darüber diskutiert, dass man erledigte Spiele in der Player-Ansicht gar nicht mehr anzeigt.
Allerdings könnte es evtl. mehrere Spiele in der Zukunft geben, darum der Button "aktuelles Spiel", allerdings müsste man hier noch prüfen, ob das schon so wie gewünscht funktioniert oder ob einfach das neueste Spiel angezeigt wird.

**Comment by wzolleis** [comment]
Kannst du hier eine Beschreibung des Problems und des gewünschten Verhaltens eintragen ?
Der Titel alleine reicht nicht, um zu verstehen, was damit gemeint ist.

**Comment by rbbuy** [comment]
..because it makes no sense to enter status for appointments in the past

**Comment by TimZolleis** [comment]
> ..because it makes no sense to enter status for appointments in the past

True, it doesnt make sense to enter a status for appointments in the past (which is aswell prohibited through the token-authentication) but it would make sense to get a glance at appointments in the past to see if there's a history of "no's" on certain dates or weeks --> or just to see when the last appointment occured.

**Comment by wzolleis** [comment]
Die aktuelle Version zeigt in der Spiele-Ansicht keine Termine aus der Vergangenheit an.
